src/mame/drivers/umatic.cpp

```diff
@@ -0,0 +1,92 @@
+// license:GPL-2.0+
+// copyright-holders:Felipe Sanches
+/****************************************************************************
+
+    Skeleton driver for Sony U-Matic Videocassete Recorder
+
+****************************************************************************/
+
+#include "emu.h"
+#include "cpu/z80/z80.h"
+#include "machine/z80ctc.h"
+
+class umatic_state : public driver_device
+{
+public:
+	umatic_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_ctc(*this, "ctc")
+	{
+	}
+
+	void umatic(machine_config &config);
+  uint8_t io_read(offs_t offset);
+  void io_write(offs_t offset, uint8_t data);
+
+private:
+	void mem_map(address_map &map);
+	void io_map(address_map &map);
+
+	required_device<z80_device> m_maincpu;
+  required_device<z80ctc_device> m_ctc;
+};
+
+void umatic_state::io_write(offs_t offset, uint8_t data)
+{
+  //FIXME!
```

Comment:
This file is still using a mixture of spaces and tabs for indentation.

Reply:
I think I fixed all the indentation issues. Let me know if I still missed any of them.